src/caselawclient/xquery/sensitive.xqy

```diff
@@ -0,0 +1,8 @@
+xquery version "1.0-ml";
+
+declare variable $uri as xs:string? external;
+declare variable $sensitive as xs:string? external;
+
+let $props := ( <sensitive>{xs:boolean($sensitive)}</sensitive> )
```

Comment:
Same here with the external variable, but I'm not sure if that plays nicely with the element tag in this one? Worth a go though.